lua/nvim-tree/commands.lua

```diff
@@ -134,6 +134,16 @@ local CMDS = {
       api.tree.collapse_all(true)
     end,
   },
+  {
+    name = "NvimTreeOpenLog",
+    opts = {
+      desc = "nvim-tree: open log file",
+      bar = true,
+    },
+    command = function()
+      vim.cmd("tab :drop" .. require("nvim-tree.log").path)
```

Comment:
Not all vim users use tabs.

We could open the file as per user preferences, something like: https://github.com/nvim-tree/nvim-tree.lua/blob/ace64228ad5d89035fbe6f85e7f45a1f7b9e29c1/lua/nvim-tree/keymap-legacy.lua#L410C17-L410C21

Alternatively, add API, maybe:

```help
==============================================================================
 6.9 API DEVELOPER                                         *nvim-tree-api.dev*     

dev.open_log({opts})                            *nvim-tree-api.dev.open_log()*
    Open the log file, if logging has been enabled.

    Parameters: ~
      • {opts} (table) optional parameters

    Options: ~
      • {cmd} (string|nil) |vim.cmd()| to execute, defaults to `edit` on nil
```

Reply:
Neovim fallbacks to a current buffer automatically if `showtabline=0`.

Reply:
Please don't resolve conversations that you have not originated. It is up to reviewer to decide whether their review comment/suggestion got resolved or not.

Reply:
@gegoune It's going to be outdated anyway.

Reply:
I really wished you haven't done that.